module/zfs/dmu_object.c

```diff
@@ -341,4 +384,10 @@ EXPORT_SYMBOL(dmu_object_free);
 EXPORT_SYMBOL(dmu_object_next);
 EXPORT_SYMBOL(dmu_object_zapify);
 EXPORT_SYMBOL(dmu_object_free_zapified);
+
+/* BEGIN CSTYLED */
+module_param(dmu_object_alloc_chunk_shift, int, 0644);
+MODULE_PARM_DESC(dmu_object_alloc_chunk_shift,
+	"CPU-specific allocator grabs 2^N objects at once");
```

Comment:
It would go back to the global allocator at the "wrong" time, potentially resulting in multiple allocators hitting the same block concurrently, but this situation should rectify itself after one iteration.  But if you're concerned, is there a way to make it only settable at boot time?